### PR TITLE
Translation-ja: Add product-owner/ja/06-closing-and-contact

### PR DESCRIPTION
**Pull request by ystk**
This PR is a part of #407 . Also, this is the final PR for Japanese translations for the main articles in all segments. :)

### CODE REVIEW

File: product-owner/ja/06-closing-and-contact-article-ja.asciidoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+== おわりに
+
+是非、 http://innersourcecommons.org/[innersourcecommon.org] まで連絡をいただき、オンラインでご参加ください。
+私たちは、80以上の企業が参加するとてもアクティブなコミュニティです。
+それらの多くは、Fortune 500の企業です。
+
+他のビデオをまだ視聴されていなければ、是非ご覧ください。
+そこには、InnerSourceのプロセスをあなたの会社に実装するために必要な最初のいくつかのステップが含まれています。
+また、Trusted Committerなどの役割や、あなたの最初のコントリビューションの合意形成のような、いくつかの文書についても説明しています。
```
Comment (johtani) [inline product-owner/ja/06-closing-and-contact-article-ja.asciidoc:9]:
```suggestion
また、https://innersourcecommons.org/learn/learning-path/trusted-committer/01[Trusted Committer] などの役割や、あなたの最初のコントリビューションの合意形成のような、いくつかの文書についても説明しています。
```

**Review by johtani — CHANGES_REQUESTED** [review]
@ystk I left a suggestion, please take a look.

File: product-owner/ja/06-closing-and-contact-article-ja.asciidoc (hunk 1 above)
Comment (ystk) [reply product-owner/ja/06-closing-and-contact-article-ja.asciidoc]: Thanks for fixing this.

**Comment by ystk** [comment]
I merged your suggested change. Thank you, @johtani .

**Comment by ystk** [comment]
Hi @johtani ,
When you have a time, please review this PR.

**Review by johtani — APPROVED** [review]
LGTM!

**Review by lenucksi — APPROVED** [review]
Thanks for creating this @ystk and the review @johtani! :tada: :rocket:

**Comment by rrrutledge** [comment]
Nice work, everyone!

**Pull request merged**